src/Nri/Colors.elm

```diff
@@ -66,6 +102,56 @@ import Css exposing (hex, rgba)
 
 {-|
 
+<p style="font-size:2em; color: #00cbeb">#00cbeb</p>
+
```

Comment:
hmm... I guess I'm not sure if the descriptions should go here in the docs, or just in our StyleGuide in the main repo.